airbyte-integrations/connectors/source-lexoffice/source_lexware/spec.yaml

```diff
@@ -0,0 +1,59 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Lexware Spec
+  type: object
+  required:
+    - apikey
+  properties:
+    apikey:
+      type: string
+      description: Lexware API Key
+      title: API Key
+      pattern: "^[a-zA-Z0-9._-]*$"
+      description: Lexoffice API Key. See the <a href="https://developers.lexoffice.io/docs/#lexoffice-api-documentation">docs</a>
+        for more information on how to obtain this key.
+      airbyte_secret: true
+    start_date:
+      title: Start Date
+      type: string
+      description: UTC date and time. Any data created after this date will be replicated.
+        If this parameter is not set, all data will be replicated.
+      pattern: "^([0-9]{4}-[0-9]{2}-[0-9]{2})?$"
+      examples:
+      - '2020-12-01'
+    voucher_type:
+      title: Voucher Type
+      description: May contain a comma separated list of voucher types, or the value "any". In the latter case, all voucher types currently available in lexoffice will be returned.
+      type: string
+      pattern: "^(^$)|(^([a-zA-Z]+)(,[ ]*[a-zA-Z]+)*$)"
+      examples:
+      - 'any'
+      - 'salesinvoice'
+      - 'salescreditnote'
+      - 'purchaseinvoice'
+      - 'purchasecreditnote'
+      - 'invoice'
+      - 'creditnote'
+      - 'orderconfirmation'
+      - 'quotation'
+      - 'downpaymentinvoice'
+      - 'deliverynote'
+    voucher_status:
+      title: Voucher Status
+      description: May contain a comma separated list of voucher status, or the value "any". In the latter case, all voucher states currently available in lexoffice will be returned.
+      type: string
+      pattern: "^(^$)|(^([a-zA-Z]+)(,[ ]*[a-zA-Z]+)*$)"
+      examples:
+      - 'any'
+      - 'draft'
+      - 'open'
+      - 'overdue'
+      - 'paid'
+      - 'paidoff'
+      - 'voided'
+      - 'transferred'
+      - 'sepadebit'
+      - 'accepted'
+      - 'rejected'
```

Comment:
Why adding this and not retrieve all vouchers?

Reply:
Because the vouchers endpoint contains all types of accounting documents (invoices, quotations, confirmations, etc.). This seems counter-intuitive, but that's how the API is designed. Some used might however only be interested in invoices but not in quotations. Also, the state is important, since the user might only want to synchronize accepted quotations, since draft once are often subject to deletion.